src/encoded/types/file.py

```diff
@@ -456,6 +456,67 @@ def donors(self, request, registry, dataset, root, replicate=None):
         # Return a unique list.
         return list(set(calculated_donors))
 
+    @calculated_property(schema={
+        "title": "Origin batches",
+        "description": "The origin batch biosample(s) associated with this file.",
+        "comment": "Do not submit. This field is calculated through the derived_from relationship back to the raw data.",
+        "type": "array",
+        "items": {
+            "title": "Origin batch",
+            "description": "Accession of the origin batch biosample",
```

Comment:
if switching to @id, should update the description to say @id instead of accession